.travis.yml

```diff
@@ -39,10 +41,11 @@ install:
   - npm install -g jsonlint
 
 script:
-  - find . -type f -name "*.json" -print0 | xargs -0 -n1 jsonlint -q
-  - find . -type f -name "*.php" -print0 | xargs -0 -n1 php -l
-  - composer validate
+  - if [[ "$SNIFF" == "1" ]];then find . -type f -name "*.json" -print0 | xargs -0 -n1 jsonlint -q; fi
+  - if [[ "$SNIFF" == "1" ]];then find . -type f -name "*.php" -print0 | xargs -0 -n1 php -l; fi
+  - if [[ "$SNIFF" == "1" ]];then composer validate; fi
   - travis_wait composer install --no-interaction --no-progress --no-scripts --no-suggest --optimize-autoloader --prefer-dist --verbose
-  - if [[ "$SNIFF" == "1" ]]; then composer install-codestandards; fi
-  - if [[ "$SNIFF" == "1" ]]; then ./vendor/bin/phpcs; fi
```

Comment:
By removing this, you are removing the CS check on the code of the plugin itself. Was that intentional ?

Reply:
Fixes by running it together with the PHPCompatibility standard

Reply:
Nope, sorry, you've made things worse.